src/unlimited.cpp

```diff
@@ -46,15 +46,15 @@ bool IsTrafficShapingEnabled();
 
 bool MiningAndExcessiveBlockValidatorRule(const unsigned int newExcessiveBlockSize, const unsigned int newMiningBlockSize)
 {
-    // The mined block size must not be greater then the excessive block size.
-    return (newExcessiveBlockSize < newMiningBlockSize);
+    // The mined block size must be less then or equal too the excessive block size.
```

Comment:
The comment and the code are correct, it is just that one is in reverse notation which has caused confusion. I shall make them both read left to right.